docs/DeveloperGuide.md

```diff
@@ -268,7 +268,8 @@ _{Explain here how the data archiving feature will be implemented}_
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
```

Comment:
Should we Update Target User Profile to the one discussed in Week 4?

Nus CS TAs who are adept and prefer CLI over GUI, and have to keep track of their tutorial students’ contact and progress.